modules/rostests/apitests/atl/CAtlFileMapping.cpp

```diff
@@ -6,8 +6,11 @@
  */
 
 #include <atlfile.h>
+#ifndef __REACTOS__
+    #include <cstddef>
```

Comment:
Is this file included by normal ATL?
Can we otherwise test if it is already included, instead of hanging this on `__REACTOS__` ?

Reply:
Below, `offsetof` macro needs this.

Reply:
I was checking where `offsetof()` was used in ATL library code: https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/lib/atl/atlcom.h;hb=453e5bc2afacf8c9595a50a61714febf21636135#l627
I don't know whether manually defining this macro can be done ?

Reply:
Let's not go there unless we absolutely have to.

Reply:
OK.